comparisons/matrix_RE.txt

```diff
@@ -125,6 +125,9 @@ TTbar13nanoEDMin2017wf1325p8 1325.8_TTbar_13_94X*NanoAOD*NANOEDM*/step2.root DQM
 TTbar13nanoEDMin2017wf1325p9 1325.9_TTbar_13_92X*NanoAOD*NANOEDM*/step2.root DQM
 TTbar13nanoEDM106Xv1in2017wf1325p81 1325.81_TTbar_13_106Xv1NanoAOD*NANOEDMMC2017*/step2.root DQM
 ZEE13nanoEDMin2016wf1329p1 1329.1_ZEE_13_80X*NanoAOD*NANOEDM*/step2.root DQM
+TTbarmc106Xul16v2nanowf2500p310 2500.31_mc106Xul17v2+TTbarMINIAOD*/step2.root NANO
+TTbarmc106Xul17v2nanowf2500p311 2500.311_mc106Xul17v2+TTBarMINIAOD*/step2.root NANO
+TTbarmc106Xul18v2nanowf2500p312 2500.312_mc106Xul17v2+TTbarMINIAOD*/step2.root NANO
```

Comment:
```suggestion
TTbar13PU106Xul16v2nanowf2500p31 2500.31_mc106Xul17v2+TTbarMINIAOD*/step2.root NANO
TTbar13PU106Xul17v2nanowf2500p311 2500.311_mc106Xul17v2+TTBarMINIAOD*/step2.root NANO
TTbar13PU106Xul18v2nanowf2500p312 2500.312_mc106Xul17v2+TTbarMINIAOD*/step2.root NANO
```
`mc` was not used before in the short names

Should the wf name be fixed at some point? `mc106Xul17v2` is in all 3 directory names while it should be 16,17,18, respectively. Thanks for making the map correct though

Reply:
there's a typo in the matrix wf name indeed. too cumbersome to correct ATM.
this mapping is evermore cumbersome IMO. a better way should be found to ease maintenance